### DISCUSSION TITLE
"-nullify" should be enabled by default

### DISCUSSION THREAD

**Discussion by zalastone** [Purfview/whisper-standalone-win#268] [General]
it works great,I tested for the last two weeks and it made a big improvement 

"faster-whisper-xxl.exe --output_dir source --output_format vtt --model large-v2 --task translate --language ko  --patience 1.8 --prompt_reset_on_no_end 1 --reprompt 2 -nullify --ff_mdx_kim2 --compute_type float16"

**Comment by Purfview** [comment]
It's "For dev experiments", in short that means not for the users to touch.

I didn't noticed any improvements from non-speech nullification, you are experiencing improvement from `pyannote_v3` VAD which is activated with it.

Btw, I've deleted your misleading comment about a model conversion, that Norwegian model is already converted there.

**Reply by zalastone** [reply]
Sure, thank you for your hard work
